Test/Platform/Tools/MFTestSystem/TestSystem.cs

```diff
@@ -61,12 +61,12 @@ static void Main(string[] args)
                 if (args[0] == "/?")
                 {
                     DisplayHelp();
-                    return;
+                    return -1;

```

Comment:
To enable more useful troubleshooting, lets define an enumeration with a unique identifier for each error case. Furthermore we should generate a text error message on Console.Error as well.